pkg/cnb/create_builder.go

```diff
@@ -65,6 +65,8 @@ func (r *RemoteBuilderCreator) CreateBuilder(ctx context.Context, builderKeychai
 		builderBldr.AddGroup(buildpacks...)
 	}
 
+	builderBldr.additionalLabels = spec.AdditionalLabels
```

Comment:
All the other steps are modifications via setters on the builderBldr. Can we add a set additional labels?

Reply:
happy to, do you think it adds value though?

I guess said differently I'm probably going to have that setter effectively is going to say exactly what I say here - which may be because I'm not writing good go code or something